cfme/common/vm_console.py

```diff
@@ -62,10 +62,27 @@ def get_screen(self):
             https://qxf2.com/blog/selenium-html5-canvas-verify-what-was-drawn/
             https://stackoverflow.com/questions/38316402/how-to-save-a-canvas-as-png-in-selenium
         """
+        # Internal function to use in wait_for().   We need to try to get the
+        # canvas element within a try-catch, that is in within a wait_for() so
+        # we can handle it not showing up right away as it is wont to do on
+        # at least RHV providers.
+        def _get_canvas_element(provider):
+            try:
+                canvas = provider.get_remote_console_canvas()
+            except ItemNotFound as e:
+                logger.info('Could not find canvas element.')
```

Comment:
If you use logger.exception here, you don't have to capture the exception (` as e`), and it'll automatically include the exception information with your given log message